src/middlewared/middlewared/plugins/config.py

```diff
@@ -206,6 +206,8 @@ def move(src, dst):
 
         self.middleware.call_hook_sync('config.on_upload', UPLOADED_DB_PATH)
 
+        # make sure we includ the uploaded db to be sent to the standby controller
+        upload.insert(0, UPLOADED_DB_PATH)
```

Comment:
Weird that I did not do that, just replace `shutil.move` with `move` in line 186 above